ts/test/transpiler_tests.bzl

```diff
@@ -132,35 +117,6 @@ def transpiler_test_suite():
         ],
     )
 
-    # This target proves that transpilation doesn't require typechecking:
```

Comment:
This test is essentially moved to `e2e/test/validation.bats` because it needs to be run with `--norun_validations`  for it to preserve the existing functionality.